supporter-product-data/src/main/scala/com/gu/services/ConfigService.scala

```diff
@@ -64,9 +65,14 @@ class ConfigService(stage: Stage) extends StrictLogging {
           "2c92a0ff56fe33f301572314aed277fb",
           "2c92a0fc610e738901612d83fce461fd",
           "2c92a0fe56fe33ff015723143e4778be",
+          "8a1295918021d0d2018022d4ca0c4aac",
```

Comment:
Found some more rate plans to exclude from the extract. We should probably have a list of rate plans to include rather than one of excluded rate plans.

Reply:
yeah what a pain!